src/AlgebraicGeometry/Schemes/CoveredSchemes/Objects/Properties.jl

```diff
@@ -32,16 +32,35 @@ Return the boolean value whether a covered scheme `X` is smooth.
 is_smooth(X::AbsCoveredScheme) = is_smooth(underlying_scheme(X))
 
 @attr function is_smooth(X::CoveredScheme)
-  if !isdefined(X, :coverings) 
+  if !isdefined(X, :coverings)
     return true
   end
   # TODO: Can this be optimized? We only need to check the rank of
-  # the jacobian matrices where we haven't checked in another chart before. 
+  # the jacobian matrices where we haven't checked in another chart before.
   # This is a tradeoff between cost of Jacobian criterion and cost of
   # optimizing the use of the covering
   return all(is_smooth, affine_charts(X))
 end
 
+# Use decomposition_info
+function _jacobian_criterion(X::CoveredScheme{<:Field})
```

Comment:
@HechtiDerLachs 
Is a decomposition info guaranteed to exist and to be set?

Reply:
According to its docstring, no.